### ISSUE TITLE
[chectl] there are nightly tags in yamls of chectl:7.12.0 tarball

### ISSUE THREAD

**Issue by dmytro-ndp** [eclipse-che/che#16769]
### Describe the bug
There are nightly tags used in:
- [7.12.0 default custom resource yaml](https://github.com/eclipse/che-operator/blob/7.12.0/deploy/crds/org_v1_che_cr.yaml#L22).
- [7.12.0 operator.yaml](https://github.com/eclipse/che-operator/blob/7.12.0/deploy/operator.yaml#L27)

At the same time [chectl:7.12.0 tar ball](https://github.com/che-incubator/chectl/releases/tag/20200424063811) operator installer is installing _Eclipse Che 7.12.0_ version using **quay.io/eclipse/che-operator:7.12.0** by default:
```
chectl server:start --listr-renderer=verbose --self-signed-cert --debug --multiuser --installer=operator --platform=openshift -n ndp-che-7-12-0
``` 

### Che version

 - [ ] latest
 - [ ] nightly
 - [ ] other: 7.12.0 release version

### Runtime
   - [ ] kubernetes (include output of `kubectl version`)
   - [ ] Openshift (include output of `oc version`)
   - [x] minikube (include output of `minikube version` and `kubectl version`)
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

### Installation method
   - [ ] chectl 
   - [x] che-operator 
   - [ ] minishift-addon
   - [ ] I don't know

### Environment
   - [ ] my computer
       - [ ] Windows
       - [ ] Linux
       - [ ] macOS
   - [ ] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify)
   - [ ] other: please specify

**Comment by tolusha** [comment]
nightly tag in https://github.com/eclipse/che-operator/blob/7.12.0/deploy/operator.yaml#L27

**Comment by mkuznyetsov** [comment]
Fixed for future releases starting from 12.1. Please reopen the issue if this reoccurs later at any point

**Comment by nickboldt** [comment]
Or add to https://github.com/eclipse/che/issues/16820 if we need to fix something else in 7.12.2.

**Comment by dmytro-ndp** [comment]
@mkuznyetsov: reopening an issue, because there are still **nightly** tags instead of release tags 7.12.1 in:
- [7.12.1 default custom resource yaml](https://github.com/eclipse/che-operator/blob/7.12.1/deploy/crds/org_v1_che_cr.yaml#L22);
- [7.12.0 operator.yaml](https://github.com/eclipse/che-operator/blob/7.12.1/deploy/operator.yaml#L27).

**Comment by tolusha** [comment]
Fixed in 7.12.x branch
https://github.com/eclipse/che-operator/blob/7.12.x/deploy/operator.yaml#L27
The next version 7.12.2 will contain proper tags/
